frontend/appflowy_flutter/lib/mobile/presentation/setting/appearance_setting_group.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/util/theme_mode_extension.dart';
+
+import 'package:appflowy/workspace/application/settings/appearance/appearance_cubit.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:go_router/go_router.dart';
+import 'setting.dart';
+
+class AppearanceSettingGroup extends StatefulWidget {
+  const AppearanceSettingGroup({
+    super.key,
+  });
+
+  @override
+  State<AppearanceSettingGroup> createState() => _AppearanceSettingGroupState();
+}
+
+class _AppearanceSettingGroupState extends State<AppearanceSettingGroup> {
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return BlocSelector<AppearanceSettingsCubit, AppearanceSettingsState,
+        ThemeMode>(
+      selector: (state) {
+        return state.themeMode;
+      },
+      builder: (context, themeMode) {
+        return MobileSettingGroup(
+          groupTitle: LocaleKeys.settings_menu_appearance.tr(),
+          settingItemList: [
+            MobileSettingItem(
+              name: LocaleKeys.settings_appearance_themeMode_label.tr(),
+              trailing: Row(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  Text(
+                    themeMode.labelText,
+                    style: theme.textTheme.bodyMedium?.copyWith(
+                      color: theme.colorScheme.onSurface,
+                    ),
+                  ),
+                  const Icon(Icons.chevron_right)
+                ],
+              ),
+              onTap: () {
+                showModalBottomSheet(
```

Comment:
<img width="471" alt="Screenshot 2023-10-23 at 19 45 41" src="https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/b1ba5082-c7a7-4c2a-8745-c063692d0cff">
<img width="472" alt="Screenshot 2023-10-23 at 19 45 49" src="https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/9cbaa91a-2f14-4f26-b09c-8d2662c71f2f">

We should consider to use the same bottom sheet style.